elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -118,16 +122,24 @@ def prepare_tag(html_tags, index, group_info={'tag': None, 'depends': None}):
         if grading_method == 'external':
             pl.check_attribs(html_tags, required_attribs=[], optional_attribs=['correct'])
         elif grading_method in ['unordered', 'ordered']:
-            pl.check_attribs(html_tags, required_attribs=[], optional_attribs=['correct', 'indent'])
+            pl.check_attribs(html_tags, required_attribs=[], optional_attribs=['correct', 'indent', 'distractor-feedback'])
         elif grading_method == 'ranking':
-            pl.check_attribs(html_tags, required_attribs=[], optional_attribs=['correct', 'ranking', 'indent'])
+            pl.check_attribs(html_tags, required_attribs=[], optional_attribs=['correct', 'ranking', 'indent', 'distractor-feedback'])
         elif grading_method == 'dag':
-            pl.check_attribs(html_tags, required_attribs=[], optional_attribs=['correct', 'tag', 'depends', 'comment', 'indent'])
+            pl.check_attribs(html_tags, required_attribs=[], optional_attribs=['correct', 'tag', 'depends', 'comment', 'indent', 'distractor-feedback'])
 
         is_correct = pl.get_boolean_attrib(html_tags, 'correct', PL_ANSWER_CORRECT_DEFAULT)
         answer_indent = pl.get_integer_attrib(html_tags, 'indent', None)
         inner_html = pl.inner_html(html_tags)
         ranking = pl.get_integer_attrib(html_tags, 'ranking', -1)
+        distractor_info = pl.get_string_attrib(element, 'distractor-info', DISTRACTOR_INFO_DEFAULT)
+        distractor_feedback = pl.get_string_attrib(html_tags, 'distractor-feedback', None)
+
+        if (distractor_info == 'none') and (distractor_feedback is not None):
+            raise Exception("distractor-info must be either 'show-distractors' or 'show-feedback' for the distractor_feedback tag to be used in <pl-answer>")
```

Comment:
`distractor_feedback` -> `distractor-feedback`, also change the message to talk about the new `distractor-info` values.

Reply:
Fixed.